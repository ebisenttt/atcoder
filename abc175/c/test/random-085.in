239368142740830 2450762968423 313763778143068
