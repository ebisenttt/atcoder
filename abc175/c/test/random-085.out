74395635402238
